src/components/bcros/filing/common/HeaderActions.vue

```diff
@@ -310,6 +315,16 @@ const showCommentDialog = (show?: boolean) => {
   isCommentOpen.value = show
 }
 
+const goToNoticeOfWithdrawal = () => {
+  const businessIdentifier = isBootstrapFiling ? tempBusinessIdentifier.value : currentBusinessIdentifier.value
+  const path = `/${businessIdentifier}/notice-of-withdrawal/`
+  const params = {
+    filingToBeWithdrawn: filingId.value.toString(),
+    filingId: '0'
+  }
+  goToBusinessDashboard(path, params)
```

Comment:
Per Arwen's PR, this is being renamed to `goToFilingUI`.

Basically, after Arwen merges her ticket, and after you rebase, verify that this is updated.